File: docs/api/get-bulk/request.js
// Get-Bulk RustOpus example

import { writeFile } from "fs/promises";

const AUTHCODE = "AAAAAAAA-0000-0000-0000-BBBBBBBBBBBB";
const PID = 1234;
const OUTFILE = "get-bulk.xml";

const response = await fetch(`<url-to-rustopus>/get-images?authcode=${AUTHCODE}&pid=${PID}`);
const text = await response.text();

await writeFile(OUTFILE, text);

File: docs/api/get-bulk/request.cs
// Get-Bulk RustOpus example
using System.Net.Http;

var authcode = "AAAAAAAA-0000-0000-0000-BBBBBBBBBBBB";
var pid = 1234;
var outFile = "get-bulk.xml";

using var client = new HttpClient();
var response = await client.GetStringAsync($"https://<url-to-rustopus>/get-bulk?authcode={authcode}&pid={pid}");
await File.WriteAllTextAsync(outFile, response);
